src/components/internals/multivalue/multivalue.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { css } from '@emotion/core';
+import PropTypes from 'prop-types';
+import Tag from '../../tag';
+
+const MultiValue = props => {
+  return (
+    <div
+      css={css`
+        display: inline-block;
+        margin: 2px;
+      `}
+    >
+      <Tag
+        styles={{
+          body: css`
+            padding: 4px 8px 3px 6px;
```

Comment:
I'm not sure of a better way to do this. Basically our tag needs to be a little bit "slimmer", so that it doesn't make the select input 1 pixel taller 🤔

Reply:
Is it only because of this that you had to add that `styles.body` prop to the Tag?

Reply:
Hm. Maybe do math on the vars instead so it's at least able to adjust?

Reply:
@jonnybel yeah 😄

Reply:
Is it possible that we can add a wrapper to the Tag, center the Tag inside it, and with `overflow: hidden` trim the excess?

Reply:
wouldn't we lose the border in that case though? 🤔

Reply:
another approach would just be to use the big Tag and let design approve it 🤔

Reply:
Yeah you're right.

One question: is there a reason to strictly set the vertical paddings of the body of the Tag, instead of setting a fixed height and let the flexbox align the text to the center?